### PR TITLE
Add sparknacl-4.0.0.toml

### PR DESCRIPTION
**Pull request by docandrew**
Added latest SPARKNaCl release

### CODE REVIEW

File: index/sp/sparknacl/sparknacl-4.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description = "Verified SPARK 2014 re-implementation of TweetNaCl cryptographic library"
+long-description = "SPARK 2014 re-implementation of TweetNaCl cryptographic library, with fully automated proofs of type safety and some correctness properties"
+version = "4.0.0"
+tags = ["spark", "cryptography", "security", "nacl", "curve25519", "ed25519", "tweetnacl"]
+licenses = "BSD-3-Clause"
+website = "https://github.com/rod-chapman/SPARKNaCl"
+
+authors = ["Rod Chapman"]
+maintainers = ["Rod Chapman <rod@proteancode.com>"]
+maintainers-logins = ["rod-chapman"]
```
Comment (mosteo) [inline index/sp/sparknacl/sparknacl-4.0.0.toml]: Hi @docandrew, if you're taking over maintainership from @rod-chapman (or co-maintaining), these fields should be updated accordingly.

**Review by mosteo — CHANGES_REQUESTED** [review]

File: index/sp/sparknacl/sparknacl-4.0.0.toml (hunk 1 above)
Comment (rod-chapman) [reply index/sp/sparknacl/sparknacl-4.0.0.toml]: I am happy to remain as maintainer...

Comment (rod-chapman) [reply index/sp/sparknacl/sparknacl-4.0.0.toml]: What do I need to do to add docandrew as an additional maintainer to allow this PR to progress?

Comment (docandrew) [reply index/sp/sparknacl/sparknacl-4.0.0.toml]: I was just submitting this to be helpful but can modify the PR to add my name if @rod-chapman is OK with it.

Comment (mosteo) [reply index/sp/sparknacl/sparknacl-4.0.0.toml]: With Jon edition, things are fine, thanks.

**Pull request merged**